wabt/src/main/java/com/dylibso/chicory/wabt/Wast2Json.java

```diff
@@ -33,6 +33,12 @@ public boolean isLoggable(Logger.Level level) {
                     return false;
                 }
             };
+    private static final com.dylibso.chicory.wasm.Module wasmModule =
+            Module.builder(Wast2Json.class.getResourceAsStream("/wast2json"))
+                    .withInitialize(false)
+                    .withStart(false)
+                    .build()
+                    .wasmModule();
```

Comment:
I think this would be the way to go

Reply:
Completely agree.
This will allow for maximum reuse and e.g. swapping `HostImports` for different `Module` executions with WASI.

side note:
I keep being confused by:
`com.dylibso.chicory.wasm.Module`
`com.dylibso.chicory.runtime.Module`
but I don't have better names to differentiate too :cry: